ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c

```diff
@@ -1778,6 +1778,112 @@ bdb_instance_start(backend *be, int mode)
     return return_value;
 }
 
+static int
+bdb_get_page_count(dbi_db_t *db, dbi_txn_t *txn, uint32_t *count)
```

Comment:
Always called with NULL txn, can we get rid of it ?

Reply:
> Always called with NULL txn, can we get rid of it ?

Yeah we can drop it.  In early development it was in place where it was being set, but not anymore.